crates/primitives/Cargo.toml

```diff
@@ -81,10 +82,9 @@ pprof = { workspace = true, features = [
     "frame-pointer",
     "criterion",
 ] }
-secp256k1.workspace = true
 
 [features]
-default = ["c-kzg", "alloy-compat", "std", "reth-codec"]
+default = ["c-kzg", "alloy-compat", "std", "reth-codec", "secp256k1"]
```

Comment:
we no longer enable c-kzg on workspace level, we need to be extra careful here and double check that we enable this somewhere.

I'm actually not sure where we enable revm c-kzg so we need to check and centralize these crucial settings, ideally on the node level